math/wxMaxima/Portfile

```diff
@@ -4,37 +4,49 @@ PortSystem          1.0
 
 PortGroup           active_variants 1.1
 PortGroup           cmake 1.1
+PortGroup           compiler_blacklist_versions 1.0
 PortGroup           wxWidgets 1.0
 PortGroup           github 1.0
 
-github.setup        wxMaxima-developers wxmaxima 21.02.0 Version-
-revision            1
+github.setup        wxMaxima-developers wxmaxima 23.05.1 Version-
+revision            0
+checksums           rmd160  195fbba20e3cd1b83e19b3e3b76f1ea8f60e267c \
+                    sha256  0c26d1439f5aba30cae94d5718d9c35bf83490953cc8696c371658ed13fd57c1 \
+                    size    16220058
+
 name                wxMaxima
 maintainers         @MSoegtropIMC
 license             GPL-2
 # openssl is a dependency of curl in cmake, so build only
 license_noconflict  openssl
-platforms           darwin
 categories          math aqua
 description         Graphical user interface for Maxima based on wxWidgets
 long_description    Maxima is a Computer Algebra System (CAS) and wxMaxima is a work book style \
                     graphical front end for it based on wxWidgets
 
-checksums           rmd160  1160cba3db29ab059afcd2dafbbe02425b2f83f0 \
-                    sha256  a8fd7228beeaa93ae8017d24d3bb652815ed159bd943a6e7c939643a705e3500 \
-                    size    16251280
+github.tarball_from archive
 
 universal_variant   no
 
+compiler.cxx_standard \
+                    2014
+
+# https://github.com/wxMaxima-developers/wxmaxima/issues/1784
+compiler.blacklist-append \
+                    {clang < 1316} \
+                    macports-clang-3.* \
+                    {macports-clang-[4-9].0} \
+                    {macports-clang-1[0-2]}
+
 # Note: as of Match 2020 (1b00466)the build with wxWidgets-3.2 has serious issues with scrolling
-wxWidgets.use       wxWidgets-3.0
+wxWidgets.use       wxWidgets-3.2
```

Comment:
@MSoegtropIMC What was the serious scrolling issue? With this PR, we're switching to wxWidgets 3.2, so I hope that whatever problem with wxWidgets 3.0 this comment was alluding to has been fixed in 3.2 and that this comment can be removed.

Reply:
When I last tried it, scrolling was seriously and obviously broken - if it is not seriously and obviously broken, it should be fine.

Reply:
I don't know how to use this software so I don't even know how to scroll to test if it is broken.

Reply:
As I said - I will do a test build and see if it works for me.